samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -66,6 +66,11 @@ static struct bt_iso_big_create_param big_create_param = {
 	.framing = 0, /* 0 - unframed, 1 - framed */
 };
 
+static size_t reset_timeout_counter(uint32_t interval_us, uint32_t timeout_us)
```

Comment:
size_t is represents count of bytes, it is incorrect to use as data type for counter.

Reply:
I agree. Should just use a uint32_t